src/test/scala/chiselTests/NamingAnnotationTest.scala

```diff
@@ -4,12 +4,13 @@ package chiselTests
 
 import chisel3._
 import chisel3.experimental.chiselName
+import chisel3.experimental.HasChiselName
 import chisel3.internal.InstanceId
 import chisel3.stage.ChiselStage
 
 import scala.collection.mutable.ListBuffer
 
-trait NamedModuleTester extends Module {
+trait NamedModuleTester extends Module with HasChiselName {
```

Comment:
why do we need `HasChiselName`, isn't `Module` already `HasChiselName`?

Reply:
`Module` isn't in the named components so `Module` val`s don't get seen by the compiler as one